src/EasyAbp.Abp.Trees.EntityFrameworkCore/EasyAbp/Abp/Trees/EntityFrameworkCore/DependencyInjection/EfCoreTreeRepositoryRegister.cs

```diff
@@ -9,18 +9,20 @@
 
 namespace EasyAbp.Abp.Trees.EntityFrameworkCore.DependencyInjection
 {
-    public class EfCoreTreeRepositoryRegistrar
```

Comment:
This is not typo,ref from [abp](https://github.com/abpframework/abp/blob/42f37c5ff01ad853a5425d15539d4222cd0dab69/framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/EntityFrameworkCore/DependencyInjection/EfCoreRepositoryRegistrar.cs)
I will keep `EfCoreTreeRepositoryRegister`

Reply:
OK, I'm changing it back to EfCoreTreeRepositoryRegistrar